wpeverest/everest-forms
# Fix - Required issue in multipart/row conditional

### All Submissions:






### Changes proposed in this Pull Request:



This PR allows user to add field with required in multipart/row which has conditional logic.

### How to test the changes in this Pull Request:

1. Create Form, Add row/multipart with conditional logic.
2. Add Field with required option enabled.
3. Verify form works as expected.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Required issue in multipart/row conditional
